### DISCUSSION TITLE
How to do the aggregation (hourly, daily) for the gauge metrics in prometheus.

### DISCUSSION THREAD

**Discussion by BrivinTech** [prometheus/prometheus#16317] [Q&A]
### What did you do?

I am storing the gauge metrics (storing the PM data for every 15 mins and those PM data are reset before collecting for next 15 min cycle) 

For example, active_users metric will have the value at 10:00 is 50, at 10:15 is 25..

the retention period for these data is 1 year.  In one of my use case, I want to render the chart, which should show the date for everey 1 hour or daily. How to do the aggregation query like this in the prometheus.

I am aware that, using api/v1/query API of active_users[30d:1h] will return the data for every 1h for the last 30 days. but here, prometheus doing average of 4 15-min data and mapping to 1h. but for gauge metrics, i want to do the sum of 4 15-min data. Is there any way to do this in the prometheus.? Please help on this.

### What did you expect to see?

Nono

### What did you see instead? Under which circumstances?

None

### Prometheus version

```text
2.14.0
```

**Comment by darshanime** [comment]
@BrivinTech, I think the [discussions](https://github.com/prometheus/prometheus/discussions) is a better place for questions like these, not the bug tracker. Unless there is something thing you tried that had unexpected output?

**Comment by beorn7** [comment]
Hello from the bug scrub!

As @darshanime said above, issues are not for usage questions. We'll convert this into a discussion, but generally, using the [community channels](https://prometheus.io/community/) will be a more promising approach.
